src/odemis/dataio/tiff.py

```diff
@@ -1833,6 +1837,62 @@ def _saveAsMultiTiffLT(filename, ldata, thumbnail, compressed=True, multiple_fil
             write_image(f, data[i], write_rgb=write_rgb, compression=c, pyramid=pyramid)
 
 
+def extract_imagej_metadata(ldata) -> str:
+    """
+    Create ImageJ compatible metadata that is added to the ImageDescription identifier for the tiff files.
+    :param ldata: (list of DataArray) list of 2D data of int or float. Should have at least one array
+    :return (str): metadata compatible with ImageJ
+    """
+    num_channels = 0
+    num_frames = 0  # Time frames
+    num_slices = 0  # zlevels
+
+    # Check the ZYX dimensions of the first DataArray
+    Y, X = ldata[0].shape[-2:]
+    if len(ldata[0].shape) >= 3:
```

Comment:
Note that in python it's possible to have a short code that convert from tuple of length N to a tuple with a fixed length, filled with `1` at the beginning. I think something like: `(1,) * (5 - len(s)) + s`, but it should handle cases where len(s) > 5.